web_client/views/imageViewerWidget/geojs.js

```diff
@@ -133,6 +135,7 @@ var GeojsImageViewerWidget = ImageViewerWidget.extend({
             this.setBounds({[annotation.id]: this._annotations[annotation.id]});
         }
         var featureList = this._annotations[annotation.id].features;
+        this._featureOpacity[annotation.id] = {};
```

Comment:
Should we delete the entry in `_featureOpacity` when the annotation is removed in the `removeAnnotation` method?